source/includes/administration/_organizations.md

```diff
@@ -69,7 +69,13 @@ Attributes | &nbsp;
 `environments`<br/>*Array[[Environment](#administration-environments)]* | The environments belonging to the organization.<br/>*includes*: `id`
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles belonging to the organization.<br/>*includes*: `id`
 `serviceConnections`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The services for which the organization is allowed to provision resources.<br/>*includes*: `id`,`serviceCode`
+`resourceCommitments`</br>*Array[[ResourceCommitment](#administration-retrieve-a-resource-commitment)]* | The resource commitments applied on the organization.
 `users`<br/>*Array[[User](#administration-users)]* | The users of the organization.<br/>*includes*: `id`
+`notes`<br/>*string* | Organization notes.
+`isDbAuthentication`<br/>*boolean* | Whether or not the organization supports db authentication.
```

Comment:
Should be _LDAP_ and _database_ authentication

Reply:
4cdafc3c34eaae06cf9dc2a14eda26a696e61337